src/diagnostics/structs/CommandRunResult.ts

```diff
@@ -26,6 +26,9 @@ export class CommandRunResult<ResponseType extends { error?: {}, data?: {} }> {
             const outputJson = JSON.parse(this.output);
 
             this._stdout = outputJson.stdout;
+            if (!this.error) {
```

Comment:
I don't seem to find usage of this field within this PR. I guess it's good to have, but can you explain your intention?

Reply:
If I do this, then the stderr then appears on the Output window.

Reply:
We don't really need this either way. The same information will be shown in the DEBUG CONSOLE when debugger starts. It's the payload of the first debug protocol message. Suggest remove altogether.